libs/gi/page-team/src/CharacterDisplay/Tabs/TabOptimize/Components/TargetSelectorModal.tsx

```diff
@@ -53,12 +53,19 @@ export function TargetSelectorModal({
               key,
               Object.fromEntries(
                 Object.entries(sectionObj).filter(([_sectionKey, node]) => {
-                  const { unit, variant } = resolveInfo(node.info)
+                  const { path, unit, variant } = resolveInfo(node.info)
+
                   if (flatOnly && unit === '%') return false
 
                   if (excludeHeal && variant === 'heal') return false
 
-                  if (!showEmptyTargets && node.isEmpty) return false
+                  if (
+                    !showEmptyTargets &&
+                    node.isEmpty &&
+                    (path !== 'eleMas' || key !== 'basic')
```

Comment:
Should be fine to only check if key !== basic. eleMas should already be in there afaik

Reply:
It is already there, but it also shows healing bonus and physical damage bonus. Do you want those to be shown as well?

Reply:
Yes